Dockerfile

```diff
@@ -14,7 +14,7 @@ WORKDIR /code/
 COPY . /code/
 WORKDIR /code/
 
-RUN pip install -r /code/test/example/requirements.txt
+RUN pip install -r /code/test/example/requirements_dev.txt
```

Comment:
there are two files now.

Parsing `html` during tests require additional dependencies.

Reply:
That doesn't require to split the requirements file. The requirements file here is only for developing purpose and no need to split the file and increase the complexity.

Reply:
okay, so quoting the documentation from our readme here:

> You can play with the example app using local virtual environment
...
....
or run docker


How would you want the users in case want to "play" with the app. Asking them to install all the dependencies won't be a great thing in my opinion.

As of this pull request, it is asking them to install all the dependencies when using docker but not through virtual environment.

Reply:
I don't see that the dependancies is something we need to worry about, specially they are used for testing purpose and the example is in fact just for testing purpose. Most likely the one who wants to faddle with the app in the example will run the test so the dependencies will be needed.

Can you please revert the requirements changes and add the new dependencies there?